src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -7854,5 +7845,70 @@ public static void Main()
                 //         lam(5);
                 Diagnostic(ErrorCode.ERR_NoCorrespondingArgument, "lam").WithArguments("arg2", "<anonymous delegate>").WithLocation(6, 9));
         }
+
+        [Fact]
+        public void ParamsArray_Call()
+        {
+            var source = """
+                var lam = (params int[] xs) => System.Console.WriteLine(xs.Length);
+                lam();
+                lam(1);
+                lam(1, 2, 3);
+                lam(new[] { 1, 2, 3 });
+                """;
```

Comment:
>""" [](http://example.com/codeflow?start=16&length=3)

Consider also testing `lam(null)`, although that will require changing the `WriteLine()` call to avoid a `NullReferenceException`.

Reply:
Good idea, added: 0f6d4d150cb00684e393ad9f780fe05c7863b82a